index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
```

Comment:
@topeogunleye  As I told you will see the same website as Figma when you inspect element at 320 px width in Google Chrome but you do not seem to follow my instruction. Even the width of your screenshot is beyond 320px. Please recheck my website according to Figma project's width.

![web](https://user-images.githubusercontent.com/114587346/198191172-41278fb0-db9b-425f-b480-a4bdc20122c7.png)
![web2](https://user-images.githubusercontent.com/114587346/198191180-e024ec9b-3535-41a1-a66a-edca50f26db2.png)
![web3](https://user-images.githubusercontent.com/114587346/198191985-868c3dd6-3f40-44d4-b4e9-b7452c919411.png)

Reply:
> @topeogunleye As I told you will see the same website as Figma when you inspect element at 320 px width in Google Chrome but you do not seem to follow my instruction. Even the width of your screenshot is beyond 320px. Please recheck my website according to Figma project's width.
> 
> ![web](https://user-images.githubusercontent.com/114587346/198191172-41278fb0-db9b-425f-b480-a4bdc20122c7.png) ![web2](https://user-images.githubusercontent.com/114587346/198191180-e024ec9b-3535-41a1-a66a-edca50f26db2.png) ![web3](https://user-images.githubusercontent.com/114587346/198191985-868c3dd6-3f40-44d4-b4e9-b7452c919411.png)

@ohyajk, I used an iPhone 12 which has a screen width of 390 to inspect.

Reply:
I told you this is responsive. Why are  you looking for tablet width in mobile site. This comes in later stage in course I guess. you just wasted my review that's for sure...

Reply:
> I told you this is responsive. Why are you looking for tablet width in mobile site. This comes in later stage in course I guess. you just wasted my review that's for sure...

@ohyajk, Not just tablet size. `320px` would not work for any other mobile phones except `320px` phones. Why not use convert px to percentages and use that so that they remain consistent and sized appropriately for all sizes.